docs/customer/routing.md

```diff
@@ -116,6 +116,8 @@ Used for troubleshooting, you can remove carriers from a route and run a quick t
 
 + **Lock** (Allow): One or more rate cards from the list of available providers.
 + **Exclude** (Deny): Exclude access to one or more rate cards in the list of available providers.
++ **Redial Max Count**: This a smart limitation feature which allow the carrier to restrict the maximum number of times their customers can redial in a selected time frame which is **Redial Max Period**. For example, you select 5 for this field, it means your customer can dial only 5 times.
```

Comment:
Redial Max Count is a smart limitation feature that allows the carrier to restrict the maximum number of times their customers can redial. After reaching this limit, customers must wait for a certain amount of time defined by the Redial Max Period.

Reply:
corrected